tests/lib/prepare-utils.sh

```diff
@@ -137,15 +137,21 @@ install_core22_deps() {
     sudo snap install ubuntu-image --classic
 }
 
-download_core22_snaps() {
+download_core24_snaps() {
     local snap_branch="$1"
 
     # get the model
-    curl -o ubuntu-core-amd64-dangerous.model https://raw.githubusercontent.com/snapcore/models/master/ubuntu-core-22-amd64-dangerous.model
+    #curl -o ubuntu-core-amd64-dangerous.model https://raw.githubusercontent.com/snapcore/models/master/ubuntu-core-24-amd64-dangerous.model
```

Comment:
Maybe add a TODO here to get it from the models repo when we have the modle there?

Reply:
Done.